posit-dev/positron
# Update Positron icon on Windows

### Intent

Fix icon for Positron.exe on Windows
Addresses #2089

### Approach

Uses a .ico with multiple resolutions to support windows icon requirements.
Ensure assets are copied during build.
Use rcedit to replace icon on Positron.exe

### QA Notes

I built Positron.exe locally on Windows and it had the correct icon, but we will need to verify from a Windows build.
